resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -58,37 +58,8 @@ EXEC="$JAVA_BIN -XX:OnOutOfMemoryError=\"kill -9 %p\" -Djava.security.egd=/dev/u
 #
 do_start()
 {
-    <% if EZBake::Config[:open_file_limit] -%>
-    [ -n "$OPEN_FILE_LIMIT" ] && ulimit -n $OPEN_FILE_LIMIT
```

Comment:
I don't know if the code to check open files values should be left in or not; I don't see other init scripts from other vendors doing that, but nothing I checked starts java services.